altair/examples/airports_count.py

```diff
@@ -0,0 +1,48 @@
+"""
+Locations of US Airports
+========================
+This is a layered geographic visualization that shows the positions of US
+airports on a background of US states.
+"""
+# category: maps
+import altair as alt
+from vega_datasets import data
+
+airports = data.airports.url
+states = alt.topo_feature(data.us_10m.url, feature='states')
+
+# US states background
+background = alt.Chart(states).mark_geoshape(
+    fill='lightgray',
+    stroke='white'
+).properties(
+    width=500,
+    height=300
+).project('albersUsa')
+
+# airport positions on background
+points = alt.Chart(airports).transform_aggregate(
+    latitude='mean(latitude)',
+    longitude='mean(longitude)',
+    count='count()',
+    groupby=['state']
+).mark_circle().encode(
+    longitude='longitude:Q',
+    latitude='latitude:Q',
+    size=alt.Size('count:Q', title='Number of Airports'),
+    color=alt.value('steelblue'),
+    tooltip=['state:N','count:Q']
+).properties(
+    title='Number of airports in US'
+)
+
+chart = background + points 
+chart = chart.configure_title(
+    fontSize=24,
+    color='black'
+).configure_legend(
+    titleFontSize=14,
+    labelFontSize=12,
+)
+
+chart
```

Comment:
For the sake of this example, I think I'd avoid minor stylistic tweaks like these. I see the primary purpose of this example as demonstrating overlaying points on a map, and the additional configuration might be a bit distracting.

Reply:
Okay, I removed the styling part for this example. But I think it would be a great idea to add an example just to demonstrate some common styling operation. It is much easier than going through the doc. (Most of the time we just want a slightly larger Title, label, etc.)